### DISCUSSION TITLE
How to Ignore routes?

### DISCUSSION THREAD

**Discussion by techaum** [ThreeMammals/Ocelot#1764] [Q&A] [answered]
Hi,

I have a controller with some routes running in my ocelot project. I want these routes like `/login`, `/auth`, `/info` to be ignored and not pass in the Route process.

I tried add the endpoints with the same downstream/upstream paths and some manual handlers, but without success.

Any ideas?

**Comment by TomPallister** [comment]
@techaum I think you need to use MapWhen to skip the Ocelot pipeline. You should be able to hardcode whatever routes you dont want to run in Ocleot. Docs are [here](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/middleware/?view=aspnetcore-2.1&tabs=aspnetcore2x)

**Comment by techaum** [comment]
Exactly that, works like a charm! Thanks sir!

**Comment by TomPallister** [comment]
no problem :)

**Comment by Rakiah** [comment]
Could you post the code that you've used to do that ? 

I can't really seem to find the way @techaum @TomPallister

**Comment by Cogitri** [comment, marked as answer]
FWIW, if you wanted to you can do something like this:

```
app.MapGroup("/account").MapIdentityApi<User>();

app.MapWhen((ctx) => !ctx.Request.Path.StartsWithSegments("/account"), (app) =>
{
	app.UseOcelot();
});
```

That way everything that's sent to `/account` will be sent to to authentication thingie of `MapIdentityApi` and everything else will be handled by Ocelot.
